src/components/select-project/select-project.controller.ts

```diff
@@ -114,11 +113,6 @@ export class SelectProjectController implements angular.IController {
     }
   }
 
-  public onNewProjectNameChange() {
-    this.ctrl.nameTaken = false;
-    this.ctrl.forms.createProjectForm.name.$setValidity('nameTaken', !this.ctrl.nameTaken);
```

Comment:
Is there a reason not to set form validity? I suspect at least one of the dialogs relies on this.